docs_src/design/adr/0005-Service-Self-Config.md

```diff
@@ -0,0 +1,74 @@
+# Service Self Config Init & Config Seed Removal  
+
+## Status 
+
+under review - for Geneva release
+
+## Context 
+Since its debut, EdgeX has had a configuration seed service (config-seed) that, on start of EdgeX, deposits configuration for all the services into Consul (our configuration/registry service).  For development purposes, or for those resource cosntrained platforms, Consul would not be running, the config-seed would not run or if it did run would obviously not deposit configuration, and services would use a configuration file in proximity to the service in order to get their neede configuration.
+
+While this process has nominally worked for several releases of EdgeX, there has always been some issues with this extra initialization process (config-seed), not least of which are:
+- race conditions on the part of the services, as they bootstrap, coming up before the config-seed completes its deposit of configuration into Consul
+- how to deal with "overrides" such as environmental variable provided configuration overrides. As the override is often specific to a service but has to be in place for config-seed in order to take effect.
+- need for an additional service that is only there for init and then dies (confusing to users)
+
```

Comment:
I think it will help to add a paragraph about Device and App services already use self seed and env overrides in the manner proposed below.

Reply:
I added this statement to the ##Decision - as it really indicated defacto decision.